src/commands/test/lib/help-messages.ts

```diff
@@ -40,35 +42,38 @@ export module Messages {
+      export const AppiumBuildDir = "Path to the directory with the Appium tests (usually <project>/target/upload)";
       
-      export const CalabashProjectDir = 'Path to Calabash workspace directory (usually <project>/features)';
+      export const CalabashProjectDir = 'Path to the Calabash workspace directory (usually <project>/features)';
       export const CalabashSignInfo = "Use Signing Info for signing the test server";
-      export const CalabashConfigPath = "Path to Cucumber configuration file (usually cucumber.yml)";
+      export const CalabashConfigPath = "Path to the Cucumber configuration file (usually cucumber.yml)";
       export const CalabashProfile = "Profile to run. It must exist in the configuration file";
       export const CalabashSkipConfigCheck = "Force running without Cucumber profile";
       
-      export const EspressoBuildDir = "Path to Espresso output directory (usually <project>/build/outputs/apk)";
-      export const EspressoTestApkPath = "Path to *.apk file with Espresso tests. If not set, build-dir is used to discover it";
+      export const EspressoBuildDir = "Path to the Espresso output directory (usually <project>/build/outputs/apk)";
+      export const EspressoTestApkPath = "Path to the *.apk file with the Espresso tests. If not set, build-dir is used to discover it";
 
-      export const UITestsBuildDir = "Path to directory with built test assemblies (usually <project>/bin/<configuration>)";
+      export const UITestsBuildDir = "Path to the directory with the built test assemblies (usually <project>/bin/<configuration>)";
       export const UITestsStoreFilePath = "Path to the keystore file";
       export const UITestsStorePassword = 'Password to the keystore. Corresponds to the "-storepass" argument in jarsigner';
       export const UITestsKeyAlias = 'Alias to the key in the keystore. Corresponds to the "-alias" argument in jarsigner';
       export const UITestsKeyPassword = 'Password to the matching private key in the keystore. Corresponds to the "-keypass" argument in jarsigner';
       export const UITestsSignInfo = "Use Signing Info for signing the test server.";
-      export const UITestsToolsDir = "Path to directory with Xamarin UI Tests tools that contains test-cloud.exe";
+      export const UITestsToolsDir = "Path to the directory containing the Xamarin UI Tests tools including test-cloud.exe";
+
+      export const StatusTestRunId = "ID of the started test run";
+      export const StatusContinuous = "Continuously checks the test run status until it completes";
 
-      export const StatusTestRunId = "ID of started test run";
-      export const StatusContinuous = "Continuously checks test run status until it completes";
+      export const XCUITestIpaPath = "Path to the *.ipa file with the XCUITest tests";
+      export const XCUITestBuildDir = "Path to the XCUITest output directory (usually <project>/Build/Products/Debug-iphoneos)";
```

Comment:
How about just "the build output" directory?  I think "usually <project>/Build/Products/Debug-iphoneos" gives enough of a clue as to what that is..